test/utils/spec_helpers.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Generally useful helper functions for writing Jasmine unit tests.
+ */
+define([], function () {
+    'use strict';
+
+    /**
+     * Runs func as a test case multiple times, using entries from data as arguments. Like Python's DDT.
+     * @param data An object mapping test names to arrays of function parameters. The name is passed to it() as the name
+     *      of the test case, and the list of arguments is applied as arguments to func.
+     * @param func The function that actually expresses the logic of the test.
+     */
+    /* jshint loopfunc:true */
```

Comment:
I'm not sure but I think this might mess up the documentation generation (when we get to it). Can we move it into the body of the function?